CONTRIBUTING.md

```diff
@@ -8,6 +8,21 @@ We encourage you to read this project's CONTRIBUTING policy (you are here), its
 
 If you have any questions or want to read more, check out the [18F Open Source Policy GitHub repository](https://github.com/18f/open-source-policy), or just [shoot us an email](mailto:18f@gsa.gov).
 
+## Testing
+
+### Template tests
+
+The template tests are useful to see what a completed template looks like. When developing, you
+should add your expected result file in the `testdata` folder. The file should be a `.html` file
+so that when a user opens it, it will open in their browser.
```

Comment:
This is a nitpick, but: `browser` --> `web browser` (as opposed to their file browser).